Formula/php@8.0.rb

```diff
@@ -1,18 +1,26 @@
 class PhpAT80 < Formula
   desc "General-purpose scripting language"
   homepage "https://www.php.net/"
-  head "https://github.com/php/php-src.git"
+  # Should only be updated if the new version is announced on the homepage, https://www.php.net/
+  url "https://downloads.php.net/~pollita/php-8.0.0alpha1.tar.xz"
+  sha256 "2ab527b3b96908b123271ee390ea01169effeb5027a7f85dbe4d0d37d7da1628"
 
   keg_only :versioned_formula
 
+  head do
+    url "https://github.com/php/php-src.git"
+
+    depends_on "bison" => :build # bison >= 3.0.0 required to generate parsers
```

Comment:
Should we keep this outside of this section? Building a stable release from source also requires this.